src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -258,6 +264,33 @@ close()
                     std::placeholders::_1)));
 }
 
+template<class Handler, class Impl>
+void
+BaseWSPeer<Handler, Impl>::
+close(const std::uint16_t& code, const std::string& reason)
+{
+    if(! strand_.running_in_this_thread())
+        return post(strand_, std::bind(
+            // disambiguate to this close overload (also above)
+            static_cast<void(BaseWSPeer::*)(const std::uint16_t&, const std::string&)>(&BaseWSPeer::close),
+            impl().shared_from_this(),
+            code, reason));
+    do_close_ = true;
+    if(wq_.empty()){
+      boost::beast::websocket::close_reason _reason;
```

Comment:
Nit: we usually don't name variables with a leading underscore.

Reply:
Additional nit: standard indentation in this code base is four spaces.  The bulk of this pull request adheres to that quite well.  However this particular area, starting what the `if(wq_.empty())` has some indentation issues.